pkg/tfvars/aws/aws.go

```diff
@@ -244,6 +245,9 @@ func TFVars(sources TFVarsSources) ([]byte, error) {
 	if rootVolume.EBS.Iops != nil {
 		cfg.IOPS = *rootVolume.EBS.Iops
 	}
+	if rootVolume.EBS.ThroughputMib != nil {
+		cfg.Throughput = int64(*rootVolume.EBS.ThroughputMib)
+	}
```

Comment:
I think we can skip adding throughput setting here as terraform is no longer used?

Reply:
I thought I'd removed the Terraform stuff. It's gone now. We really need to remove the AWS Terraform (any any other that is still sitting around unused). I might make a card for this soon.